app/controllers/wikis_controller.rb

```diff
@@ -0,0 +1,36 @@
+class WikisController < ApplicationController
+  def index
+  end
+
+  def show
+  end
+
+  def new
+    @wiki = Wiki.new
+  end
+  
+  def create
+    @wiki = Wiki.new(wiki_params)
+    @wiki.user_id = params[:wiki][:user]
```

Comment:
Change this to: 

@wiki.user_id = params[:wiki][:user] ||= current_user.id

That way, if the user ID is not sent, we set the user to the current user. This works and passes the tests.